platform/src/Layout.js

```diff
@@ -89,7 +89,13 @@ class Layout {
 
             // Create the splitters for the row
             let splitProportions = ("10, ".repeat(numberOfRows)); // Add a proportion per splitter
-            verticalSplitters.push ( Layout.createVerticalSplitter( panelsToLayout.map( pn => pn.getElement() ), splitProportions) );
+
+            if (panelsToLayout.length > 1) {
+                verticalSplitters.push( Layout.createVerticalSplitter( panelsToLayout.map( pn => pn.getElement() ), splitProportions) );
+            } else {
+                // No splitter required for a single panel
+                verticalSplitters.push( panelsToLayout[0].getElement() );
```

Comment:
Doesn't this just create another problem on Line 107? (BTW, why **do** we create another vertical splitter there?)

Reply:
Good point, the splitter created on line 107 shouldn't be required I'll test with a single column example

Reply:
Fixed in 7ab53c7158eba08f7e2d5af13427ad60a902c85b